openshift/operator-controller/kustomize/overlays/openshift/olmv1-ns/patches/manager_deployment_certs.yaml

```diff
@@ -16,6 +16,9 @@
 - op: add
   path: /spec/template/spec/containers/0/args/-
   value: "--tls-key=/var/certs/tls.key"
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: "--catalogd-cas-dir=/var/ca-certs"
 - op: add
   path: /spec/template/spec/containers/0/env
   value: [{"name":"SSL_CERT_DIR", "value":"/var/ca-certs"}]
```

Comment:
Kinda feel like we should take this out of the manifests since we're providing a nearly-equivalent (but better) option as part of this PR.

Reply:
I'm reluctant to, since it's still being used as a base for the SystemCertPool, and thus used by container/images.

Reply:
And setting the `--pull-cas-dir` completely changes the behavior (we lose additionalTrustedCA support)
And I just realized, the SSL_CERTS_DIR is needed for running our upstream e2e testing using a local repo

Reply:
I feel like we should copy some of this out-of-codebase, out-of-mind knowledge into this file as comments to explain to our future selves:
 1. What the heck all this is for
 2. Any gotchas that we know about, so that we avoid tripping over them in the future

If folks agree with that, it'd be maybe better in a follow-up once we've got it all figured out and we're not in fire drill mode.

Reply:
There's already something documented in Google Docs.

Reply:
My main thing is having a breadcrumb _here_ that helps us make sense if what's here. Can we link the google doc here?

Reply:
Yes... but followup?